### ISSUE TITLE
Wrong time interval on server

### ISSUE THREAD

**Issue by endl3ssx** [nickfox/GpsTracker#35]
Hello,

I downloaded this repo and everything is working almost fine.
I'm using the Android Client and the PHP-MySQL server.
The problem is, even if I set the interval in the Android app to 1-5-10 minutes, the record in my MySQL DB only appear at 5 minutes intervals.
So I guess it doesn't take account of the set interval in the app.

Do you have any idea why is that happening?

Thank you!

**Comment by nickfox** [comment]
Can you possibly run the android app on your phone while using Android Studio and log how often gps data is sent (using logcat). This is a new one on me...

Logging is enabled on line 151 of this file:

https://github.com/nickfox/GpsTracker/blob/master/phoneClients/android/app/src/main/java/com/websmithing/gpstracker/LocationService.java

**Comment by endl3ssx** [comment]
I did run the logs, here is my output:

11-10 09:10:38.581 4796-4796/com.websmithing.gpstracker D/GpsTrackerActivity: startAlarmManager
11-10 09:14:05.351 4796-4796/com.websmithing.gpstracker D/LocationService: startTracking
11-10 09:14:05.401 4796-4796/com.websmithing.gpstracker D/LocationService: onConnected
11-10 09:14:09.261 4796-4796/com.websmithing.gpstracker E/LocationService: position: 46.7594925, 23.5924466 accuracy: 27.737
11-10 09:14:09.921 4796-4796/com.websmithing.gpstracker V/AsyncHttpRH: Progress 19 from 19 (100%)
11-10 09:14:09.921 4796-4796/com.websmithing.gpstracker D/LocationService: URL
11-10 09:14:09.921 4796-4796/com.websmithing.gpstracker E/LocationService: sendLocationDataToWebsite - success

As you can see, the AlarmManager was started at 09:10, and the startTracking and subsequent calls only happened at 09:14. Keep in mind that I have 1 minute intervals selected in my app.
I have no idea, the code looks perfect from my standpoint, don't really know what the issue could be.

**Comment by nickfox** [comment]
Ok, so that was the first one. Did you keep the phone and log running to get a better feel of the interval. There could be several reasons for it taking so long. In line 179, try changing accuracy to something bigger like 5000.0 and above that line, log what the accuracy actually is so that you can see it. Also make sure you are running the phone from a place that is close to a window or outdoors and has clear skies above. Having clouds, tall buildings, trees can make getting gps readings a lot worse. Run the phone for about 10 minutes to get more intervals.

**Comment by endl3ssx** [comment]
Yes, I did run it by the window and even outdoors, the position is correct, as is the distance and the speed, everything is fine. I don't think is an antenna problem, the updates happen exactly 5 minutes apart, even the seconds. I've run the app all day long today, the accuracy is fine, about 85-107, which has an impact only on my distance. I don't get why it doesn't stay on 1 minute intervals if I select them, I've seen that noone from the issues section encountered this before.

**Comment by brentfraser** [comment]
I've tested it on my Note2 (Android 4.4.2) with 1 minute interval and it works well.

**Comment by brentfraser** [comment]
#47 looks similar.
